src/hotspot/share/services/mallocLimit.cpp

```diff
@@ -25,7 +25,7 @@
 
 #include "precompiled.hpp"
 
-#include "memory/allocation.hpp"
+#include "nmt/memflags.hpp"
```

Comment:
While poking around in the includes, could you remove the blankline on 27. This style inconsistency has slowly crept into the code base.